speeches/tests/section_tests.py

```diff
@@ -176,7 +176,7 @@ def test_add_section_with_heading(self):
             'heading': 'A test section'
         })
         new_section = Section.objects.order_by('-id')[0]
-        self.assertRedirects(resp, '%s' % new_section.slug)
+        self.assertRedirects(resp, '/%s' % new_section.slug)
```

Comment:
Before 1.9 it added scheme+host (and presumably therefore the initial `/`) implicitly. All redirect checks are relative since 1.9.